app/assets/stylesheets/blacklight/_facets.scss

```diff
@@ -148,6 +148,21 @@ ul.facet_extended_list, .facet_extended_list ul
   @extend .clearfix;
 }
 
+ 
+// Facet modal alphabetical filter
+.alpha-filter {
+  margin-bottom: 18px;
+  text-align: center;
+
+  a {
+    padding: 0 2px;
+      &.active {
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented 4 spaces, but was indented 6 spaces